src/base.jl

```diff
@@ -1,5 +1,9 @@
 # This file contains code formerly part of Julia. License is MIT: https://julialang.org/license
 
+if !isdefined(Base, :something)
```

Comment:
Better check for `VERSION`, which will allow Femtocleaner to remove this block automatically. Or add this directly to Compat, which will be useful anyway.

Reply:
> Better check for VERSION, which will allow Femtocleaner to remove this block automatically. 

Good point, update incoming.

> Or add this directly to Compat, which will be useful anyway.

True. But do we want to wait for that to be included in Compat and the release published?